### ISSUE TITLE
Incorrect information in the tutorial 'Protein_Deep_Learning.ipynb'

### ISSUE THREAD

**Issue by PrasannaKasar** [deepchem/deepchem#4280]
## To Reproduce

1. if you navigate to 'examples/tutorials/Protein_Deep_Learning.ipynb', you could see under the cell of splitting train and test set, the line says 'Here we create a neuronal network using <b>tensorflow-keras</b> and using a loss function of "MAE" to evaluate the regression result.' where it shouldn't mention tensorflow-keras as deepchem has shifted its tutorials from tensorflow to pytorch

![Image](https://github.com/user-attachments/assets/5dd51071-d343-4de7-9841-1937077ebc43)

## Expected behavior

the line should say: 'Here we create a neuronal network using <b>pytorch</b> and using a loss function of "MAE" to evaluate the regression result.'

## Environment

* OS:
* Python version:
* DeepChem version:
* RDKit version (optional):
* TensorFlow version (optional):
* PyTorch version (optional):
* Any other relevant information:

**Comment by niweshsah** [comment]
Hi @PrasannaKasar,

I’ve resolved the issue and wanted to share my changes before opening a PR. Please review them when you get a chance:
🔗 [Issue-#4820 Branch](https://github.com/niweshsah/deepchem_niwesh/tree/Issue-%234820)

Let me know if any modifications are needed. Thanks!

**Comment by vishall4** [comment]
Hi, I noticed PR #4281 was opened over a year ago but the issue is still open. Is that PR still active? If not, I'd like to take this on and submit a fix
